CONTRIBUTING.md

```diff
@@ -61,6 +61,14 @@ export TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
 npm run test
 ```
 
+### Run auth integration tests
+
+```
+export TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
+export TEST_SESSION_TOKEN=<YOUR_SESSION_TOKEN>
+npm run integration-test-auth
```

Comment:
@bruuuuuuuce @rishtigupta next week we should talk about our options for making it possible to run these in CI